### PR TITLE
python3Packages.pytorch-bin: 1.7.1 -> 1.8.0

### PR DESCRIPTION
**Pull request by danieldk**
###### Motivation for this change

Changelog:

https://github.com/pytorch/pytorch/releases/tag/v1.8.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115166` at 523cd910 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/0c8ea3682a8246dc98e65d5582ed1460">python37Packages.pytorch-bin</a></li></li>
    <li><a href="https://gist.github.com/3db64138690f55692d8ee8ae307d057e">python38Packages.pytorch-bin</a></li></li>
    <li><a href="https://gist.github.com/5bf2b14ff71054615d3dabfb65a39e41">python39Packages.pytorch-bin</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/pytorch/bin.nix [hunk 1]
```diff
@@ -19,7 +19,7 @@ let
   platform = if stdenv.isDarwin then "darwin" else "linux";
```
Comment (zseri) [inline pkgs/development/python-modules/pytorch/bin.nix:19]: This variable appears to be unused.

File: pkgs/development/python-modules/pytorch/bin.nix [hunk 2]
```diff
@@ -19,7 +19,7 @@ let
...
   srcs = import ./binary-hashes.nix version;
   unsupported = throw "Unsupported system";
```
Comment (zseri) [inline pkgs/development/python-modules/pytorch/bin.nix:21]: This variable is only used once and thus could be inlined.

File: pkgs/development/python-modules/pytorch/bin.nix (hunk 1 above)
Comment (danieldk) [reply pkgs/development/python-modules/pytorch/bin.nix]: Thanks, removed!

File: pkgs/development/python-modules/pytorch/bin.nix (hunk 2 above)
Comment (danieldk) [reply pkgs/development/python-modules/pytorch/bin.nix:20]: Well, there are many variables only used once, and the line where it is used would get a bit long/tedious if it is inlined.

Comment (zseri) [reply pkgs/development/python-modules/pytorch/bin.nix:20]: ok.

**Review by fogti — APPROVED** [review]
afaik LGTM, but I haven't tested it.

**Pull request merged**